packages/core/src/components/checkbox/checkbox.scss

```
@@ -52,16 +60,17 @@
     display: flex;
 
     justify-content: flex-start;
-    align-items: flex-start;
+    align-items: center;
+    gap: var(--theme-space-0);
 
     width: 100%;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)